### ISSUE TITLE
Refactor popover module to require a unique ID for each popover instance

### ISSUE THREAD

**Issue by sebnitu** [sebnitu/vrembem#834]
## Problem

To help make the popover component more accessible, popovers should require a unique ID similar to modals and drawers. They should then also include `aria-controls="unique-id"` along with the `aria-expanded` attributes. Included in this update should be a more flexible handling of the popover API, allowing to pass the unique id for get, open and close functions.
